content/community/support.md

```diff
@@ -1,34 +1,36 @@
 ---
 id: support
-title: Where To Get Support
+title: מקורות לקבלת תמיכה
 layout: community
 sectionid: community
 permalink: community/support.html
 redirect_from:
   - "support.html"
 ---
 
-React has a community of millions of developers.
+קהילת הפיתוח של ריאקט מורכבת ממיליוני מפתחות ומפתחים.
 
-On this page we've listed some React-related communities that you can be a part of; see the other pages in this section for additional online and in-person learning materials.
+בדף זה ריכזנו קהילות בנושא ריאקט שניתן לקחת חלק בהן,ניתן גם לצפות בדפים נוספים למטה על מנת לקבל חומרים נוספים אונליין ולימוד עצמי.
```

Comment:
Maybe
בדף זה ריכזנו קהילות בנושא ריאקט שניתן לקחת חלק בהן, ניתן גם לצפות בדפים נוספים למטה על מנת לקבל חומרי לימוד עצמי.

חומרים נוספים אונליין ולימוד sounds not right

Reply:
It gets the poit across but I see your point.